forte/processors/base/batch_processor.py

```diff
@@ -252,6 +257,116 @@ def define_batcher() -> ProcessingBatcher:
         return FixedSizeDataPackBatcher()
 
 
+class Predictor(BaseBatchProcessor):
+    @staticmethod
+    def _define_context() -> Type[Annotation]:
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        return Sentence
+
+    @staticmethod
+    def _define_input_info() -> DataRequest:
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        return {}
+
+    def _prepare_coverage_index(self, input_pack: DataPack):
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        pass
+
+    def pack(self, pack: PackType, inputs) -> None:
+        r"""This function is just for the compatibility reason.
+        And it is not actually used in this class.
+        """
+        pass
+
+    @staticmethod
+    def define_batcher() -> ProcessingBatcher:
+        return FixedSizeDataPackBatcherWithExtractor()
+
+    @classmethod
+    def default_configs(cls) -> Dict[str, Any]:
+        super_config = super().default_configs()
+        super_config.update({
+            "scope": None,
+            "feature_scheme": None,
+            "batch_size": None,
+            "model": None,
+            "batcher": cls.define_batcher().default_configs()
+        })
+        return super_config
+
+    def initialize(self, resources: Resources, configs: Optional[Config]):
+        batcher_config = {}
+        batcher_config["scope"] = configs.scope
+        batcher_config["feature_scheme"] = {}
+        for tag, scheme in configs.feature_scheme.items():
+            if scheme["type"] == TrainPreprocessor.DATA_INPUT:
+                batcher_config["feature_scheme"][tag] = scheme
+        batcher_config["batch_size"] = configs.batch_size
+        configs.batcher = batcher_config
+        super().initialize(resources, configs)
+
+        assert "model" in configs
+        self.model = configs.model
+
+    def _process(self, input_pack: PackType):
+        r"""In batch processors, all data are processed in batches. So this
+        function is implemented to convert the input datapacks into batches
+        according to the Batcher. Users do not need to implement this function
+        but should instead implement ``predict``, which computes results from
+        batches, and ``pack_all``, which convert the batch results back to
```

Comment:
No need for `pack_all`? I believe you may need to update the docstrings.

Reply:
Got it.